sdk/spring/azure-spring-boot-samples/azure-spring-cloud-sample-eventhubs-binder/src/main/java/com/azure/spring/sample/eventhubs/binder/EventProducerConfiguration.java

```diff
@@ -14,7 +14,7 @@
 import java.util.function.Supplier;
 
 @Configuration
-@Profile("!manual")
+@Profile("auto")
```

Comment:
I think we can keep two profiles of manual and !manual, and remain the default profile as !manual. Otherwise we provide three profiles: auto, sync, async